### DISCUSSION TITLE
How can I disable the progess feature?

### DISCUSSION THREAD

**Discussion by andrinbr** [Eugeny/tabby#5556] [Q&A] [answered]
Hi

If I transfer a lot of small files with rsync (with -P), the terminal freezes and inputs are not possible anymore (on all tabs).
I strongly suspect the progess feature causes this issue since the terminal works fine with a lot of text output.

Using Tabby on Windows 10. 

Thanks

**Comment by Eugeny** [comment, marked as answer]
You can turn if off by setting `terminal.detectProgress: false` in the config file, but could you please capture a short performance trace from the Devtools first?

**Reply by andrinbr** [reply]
[Profile-progress.zip](https://github.com/Eugeny/tabby/files/7907235/Profile-progress.zip)

Here you go. 

Thank you for your quick reply.

**Reply by Eugeny** [reply]
I've just added a bunch of improvements to the nightly - give it a try!

**Reply by andrinbr** [reply]
I have just tried the 1.0.171-nightly and it works much better now, terminal does not freeze anymore. 

Thank you!

**Reply by TheAndrey** [reply]
Thanks, I turned it off too. I didn't like this opportunity, progress is being determined or not. The bar on the tab and the icon on the taskbar blink frequently 🙁. Performed `apt update` in WSL.

**Reply by erSitzt** [reply]
I liked this feature until i had to rsync a co... would be nice to have the "detectProgress" feature a little more accessible like in the profiles. Or at least in the tab context menu to turn it off while doing the "intense" stuff :)

**Comment by lawweiliang** [comment]
@andrin55  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
